MainDemo.Wpf/IconPackViewModel.cs

```diff
@@ -63,7 +63,7 @@ private void Search(object obj)
         private void CopyToClipboard(object obj)
         {
             var kind = (PackIconKind?)obj;
-            Clipboard.SetText($"<materialDesign:PackIcon Kind=\"{kind}\" />");
+            Clipboard.SetDataObject($"<materialDesign:PackIcon Kind=\"{kind}\" />");
```

Comment:
You will need to take it out since the PR is from your branch. The "easiest" thing to do is to simply rebase your New-control branch from master rather than from the commit with this change. 

TL;DR 
From your New-control branch Just run this command and do a force push:
git rebase --onto f1ed10c3eedd6b23f490a9f07d102c4f8cd4670b abc3228f6788c318329607
5bb033f04e6ab45c3b

For these screenshots I am using [GitKraken](https://www.gitkraken.com/) (which is free for projects like this) to show the commit tree but I am running the git commands from the command line.
Here is a view of the current commit tree:
![image](https://user-images.githubusercontent.com/952248/28327548-99e059da-6b98-11e7-96ca-2e0eba5ee58a.png)
Because we want to basically pick up the commit and change its parent we use the [Git rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) command.
The command above does just that. The `--onto <hash>` is the hash of the commit that we want to make the new parent (ie. the parent we want to be based on). The second hash is the hash of the current parent.
After running the command on the command line the tree will look like this:
![image](https://user-images.githubusercontent.com/952248/28328090-3275934e-6b9a-11e7-8e90-924928b87f2a.png)
GitKraken shows local commits with a little computer symbol, and remote commits with a little merge symbol. The remote commit will not change until you push. Because you are changing the parent you will need to [force the push](https://stackoverflow.com/questions/5509543/how-do-i-properly-force-a-git-push) to go through.
`git push origin New-control --force`
After this you should see the local and remote commits in the same spot on the commit tree.

Hope that helps.